bentoml/_internal/server/grpc/interceptors/access_log.py

```diff
@@ -0,0 +1,43 @@
+import logging
+from timeit import default_timer
+
+from opentelemetry import trace
+
+from bentoml._internal.configuration.containers import BentoMLContainer
+from bentoml._internal.server.grpc.interceptors import AsyncServerInterceptor
```

Comment:
```suggestion
from ...configuration.containers import BentoMLContainer
from . import AsyncServerInterceptor

if TYPE_CHECKING:
    from opentelemetry.trace import TraceProvider
    from ..types import BentoServicerContext
    from ..types import HandlerMethod
    from bentoml.grpc.v1.service_pb2 import Request
    from bentoml.grpc.v1.service_pb2 import Response
```
We are using relative imports here. Also it is a good practice to add types incrementally.